src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/GoalEntity.kt

```diff
@@ -56,6 +56,9 @@ class GoalEntity(
   @Column(name = "target_date")
   var targetDate: LocalDate? = null,
 
+  @Column(name = "reminder_date")
+  var reminderDate: LocalDate? = null,
```

Comment:
I think this probably needs the same JsonFormat annotation as the `createdDate` - I'm not sure why it's not on `targetDate` but I suspect that's not intentional.

Reply:
Or a version without the timestamp I guess.

Reply:
I'll update this, do we need to update the targetDate too, or could it cause problems if it hasn't been formatted as expected to date?

Reply:
I've changed this in the db from a `timestamp` to `date` because we don't care about the time. I have added the JsonFormat annotation to match too. [ed561c7](https://github.com/ministryofjustice/hmpps-sentence-plan/pull/201/commits/ed561c788900d67665d85c4514de31fce8d64b6a)